ui/widgets/nuxeo-date.js

```diff
@@ -40,7 +40,9 @@ import './nuxeo-tooltip.js';
     static get template() {
       return html`
         <span id="datetime" hidden$="[[!datetime]]">[[formatDate(datetime, format)]]</span>
-        <nuxeo-tooltip for="datetime">[[formatDateTime(datetime, tooltipFormat)]]</nuxeo-tooltip>
+        <nuxeo-tooltip for="datetime" hidden$="[[_sameFormat(datetime,format,tooltipFormat)]]">
```

Comment:
Perhaps a <dom-if> to avoid creating the tooltip in the first place as formats will likely not change.

Reply:
But if they change, the expected behavior will break right?
As soon as we stamped `nuxeo-tooltip` it will be on the DOM and acting accordingly 😅

Reply:
Just add "restamp". Not saying we shouldn't fix the "hidden"...

Reply:
As discussed with Nelson, since the hidden avoids the paper-tooltip to be created, we're ok with not changing it here to dom-if.